packages/vnf-wallet-sdk-nodejs/src/api/entities/VCLServiceType.ts

```diff
@@ -10,6 +10,7 @@ enum VCLServiceType {
     NotaryIdDocumentIssuer = 'NotaryIdDocumentIssuer',
     ContactIssuer = 'ContactIssuer',
     NotaryContactIssuer = 'NotaryContactIssuer',
+    NotaryWorkPermitIssuer = 'NotaryWorkPermitIssuer',
```

Comment:
I have two pieces of feedback - only one of which you have to do:
1. This enum shouldnt exist at all as these values come from the server side and have nothing to do with the SDK
2. But if that is too much of a change then at least ALSO add `WorkPermitIssuer`